docs/solutions/block-lan-release-solution.md

```
@@ -0,0 +1,393 @@
+
+- 将原有的区块发布修改成把区块构建产物转成二进制数据存储到数据库
+
+- 后端生成构建产物链接，替换原有区块发布后托管在unpkg的连接，格式为：http://localhost:7011/material-center/api/block/read/@opentiny-assets/tinyengine-dev-block-testceshi11-2328@1.0.17/js/web-components.umd.js
+
+- 提供链接访问获取文件的后端方法
+
+## 三、简单demo，仅供参考
+
+### 1. 新增存储数据库的表，字段如下：
+ 
+| id  	        |int(10)   	    |自增id
```

Markdownlint (0.37.0)

Column: 7
Hard tabs

(MD010, no-hard-tabs)